DDG4/edm4hep/Geant4Output2EDM4hep.cpp

```diff
@@ -25,7 +25,15 @@
 #include <edm4hep/CaloHitContributionCollection.h>
 #include <edm4hep/SimCalorimeterHitCollection.h>
 #include <edm4hep/EDM4hepVersion.h>
-#include <edm4hep/Constants.h>
+#if EDM4HEP_BUILD_VERSION <= EDM4HEP_VERSION(0, 10, 2)
+  constexpr const char* CellIDEncoding = "CellIDEncoding";
+#elif EDM4HEP_BUILD_VERSION < EDM4HEP_VERSION(0, 99, 0)
```

Comment:
```suggestion
#if EDM4HEP_BUILD_VERSION < EDM4HEP_VERSION(0, 99, 0)
```

As we don't need to support down to 0.10.2 now.

Reply:
I can take it out in my PR.